workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/PigActionNode.java

```diff
@@ -34,7 +33,6 @@ The method getXML() returns a string which contains name, Id, next success node(
 
 public class PigActionNode extends GenericActionNode {
 
-    private static final Logger LOGGER = Logger.getLogger(PigActionNode.class);
     private ProcessInfo processInfo = new ProcessInfo();
     private ActionNode actionNode = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "actionNode" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)